.github/workflows/create-release.yml

```diff
@@ -25,8 +25,8 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Validate set versions
-        run: ./validate_versions.sh ${{ github.event.inputs.name }}
+      - name: Validate the release tag
+        run: ./validate_release_tag.sh ${{ github.event.inputs.name }}
```

Comment:
Could we also just re-use the existing script from lifecycle-manager? We could checkout the lifecycle manager and invoke the script from there. Then we would not get unaligned again in the future.

Reply:
I think it's not a good practice to build dependencies for these two repos just because reusing this script. IMO, different repo should have it's own pipeline code. If certain code worth sharing, then design it as a customized github action is the right direction to go. https://docs.github.com/en/actions/sharing-automations

But the functionality of this script is simple, the efforts to publish it as github action is not worth it. And to keep the script separate is also not bad, it allows us to design different validation for both repo if needed in the future.